### PR TITLE
feat: coindcx driver

### PR DESCRIPTION
**Pull request by sajcics**

### CODE REVIEW

File: drivers/coindcx.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    const symbols = await request('https://api.coindcx.com/exchange/v1/markets_details');
+    const pairs = [];
+
+    symbols.forEach((el) => {
+      pairs[el.symbol] = {
+        quote: el.base_currency_short_name,
+        base: el.target_currency_short_name,
+        quoteName: el.base_currency_name,
+        baseName: el.target_currency_name,
+      };
```
Comment (nickpater) [inline drivers/coindcx.js:25]: Hey! These are reversed. If I look at the BTCINR market; the close they provide is the Bitcoin price in Rupees!

Comment (sajcics) [reply drivers/coindcx.js:25]: @nickpater oh, true! Updated.

Comment (nickpater) [reply drivers/coindcx.js:25]: Oh noes, I think I made a mistake. `base_currency_short_name` is INR, mh you had it correct the first time :sweat_smile:

**Pull request merged**